dmaws/commands/provision.py

```diff
@@ -0,0 +1,34 @@
+import os
+import json
+import subprocess
+
+from ..cli import cli_command
+from ..utils import run_cmd
+
+
+def run_playbook(playbook, hosts, ctx, basedir='playbooks/'):
+    args = [
+        'ansible-playbook',
```

Comment:
So (just to clarify) do we still need Ansible for 'provision' but are dropping it for create/delete/deploy?